### ISSUE TITLE
Custom file input broken and not accessible

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#25129]
Currently the custom file input https://getbootstrap.com/docs/4.0/components/input-group/#custom-file-input appears broken/unfinished. There's no indication of where the file input is
![image](https://user-images.githubusercontent.com/895831/34449101-7d0772e6-ecec-11e7-8074-f2e941a233bd.png)

Further, the file inputs seem to lack any form of labelling (no `<label for="...">` pointing to them, and no other alternative like at least an `aria-label`).

**Comment by mdo** [comment]
Duplicate of #25112.
